src/test/java/teammates/test/pageobjects/FeedbackSubmitPage.java

```diff
@@ -193,4 +193,23 @@ public void waitForOtherOptionTextToBeClickable(int qnNumber, int responseNumber
         waitForElementToBeClickable(element);
     }
 
+    public void verifyVisibilityAndCloseMoreInfoAboutEqualShareModal() {
+        WebElement moreInfoAboutEqualShareModalLink = browser.driver.findElement(By.id("more-info-equal-share-modal-link"));
+        click(moreInfoAboutEqualShareModalLink);
+        WebElement moreInfoAboutEqualShareModal = browser.driver.findElement(By.id("more-info-equal-share-modal"));
+        waitForElementVisibility(moreInfoAboutEqualShareModal);
+        closeModal();
+    }
+
+    public void closeModal() {
+        WebElement closeButton = browser.driver.findElement(By.className("close"));
+        waitForElementToBeClickable(closeButton);
+        click(closeButton);
+        try {
+            Thread.sleep(1000);
```

Comment:
@HirdayGupta Odd, I've tried replacing the entire try-catch block with `waitForElementToDisappear(By.id("more-info-equal-share-modal"));` locally, following your code snippet, and didn't run into any timeouts across my 3 test runs on that test.

As for renaming the method name, that should be fine 👍